ports/zephyr/boards/ai_m61_32s_kit.conf

```diff
@@ -0,0 +1,32 @@
+# Hardware features.
+CONFIG_PINCTRL=y
+CONFIG_GPIO=y
+CONFIG_WATCHDOG=n
+CONFIG_CONSOLE_SUBSYS=y
+CONFIG_CONSOLE_GETCHAR=y
+CONFIG_CONSOLE_GETCHAR_BUFSIZE=256
+CONFIG_I2C=y
+CONFIG_I2C_TARGET=y
+CONFIG_SPI=y
+CONFIG_MEMC=y
```

Comment:
FYI, this is already in `ai_m61_32s_kit_defconfig` here https://github.com/zephyrproject-rtos/zephyr/pull/98261/commits/07e330270b4b4a2855baf816d32c77f8f1099356
Doesn't matter.